phone-test/yarn.lock

```diff
@@ -3303,6 +3303,11 @@ babel-preset-react-app@^10.0.1:
     babel-plugin-macros "^3.1.0"
```

Comment:
Adding the yarn.lock file, though I'm aware that this practice depends on team consensus. I've worked with teams that didn't enforce the practice of adding lock files, but I think it helps ensure that the conditions, in which a feature was developed are consistent in other environments.